nemo_text_processing/text_normalization/en/data/roman/male.tsv

```diff
@@ -1608,7 +1608,6 @@ Mahmoud
 Mahmud
 Maison
 Major
-Malcolm
```

Comment:
So, this dataset essentially defines what names can possibly be followed by roman numerals, does it matter the sex for the detection, or is that used later for tagging gendered words?

I'm not sure why English would need the names separated by female/male.